.github/workflows/run-snapshot-test.yml

```diff
@@ -0,0 +1,92 @@
+name: Snapshot verification
+
+on:
+  pull_request:
+    branches: [ main, develop, feature/LD-91/golden-tests ]
+
+permissions:
+  checks: write
+  contents: write
+  statuses: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  test:
+    name: Verify snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Prepare Android Environment
+        uses: ./.github/actions/prepare-android-env
+
+      - name: Gradle - Verify snapshots with Paparazzi
+        id: testStep
+        run: ./gradlew clean components:verifyPaparazziDebug
+
+      - name: Upload snapshot failure deltas
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: snapshot-failure-deltas
+          path: /home/runner/work/Loudius/Loudius/components/out/failures/
+
+      - name: Upload snapshot failure report
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: snapshot-failure-report
+          path: /home/runner/work/Loudius/Loudius/components/build/reports/tests/testDebugUnitTest/
+
+      - name: Find PR number
+        uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+        if: always()
+        with:
+          state: open
+
+      - name: Find Comment on PR
+        uses: peter-evans/find-comment@v2
+        id: fc
```

Comment:
nitpick: this id could be a little bit more meaningful (e.g. as you have `findPr`, maybe `findComment`) but ok 👍

Reply:
Good idea 👌 I've changed it to the `findCommentId`.